lib/snippets/kubernetes-deploy

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ruby
+
+# Usage: kubernetes-deploy <app's namespace> <kube context>
+
+# Prerequisites:
+#  - kubectl 1.5.1+ binary must be available in the shipit machine's path
+#  - ENV['KUBECONFIG'] must point to a valid kubeconfig file that includes all the contexts you want to deploy to
```

Comment:
Actually, the kubeconfig is specific to the deployer (e.g. Shopify Shipit), not any group of things being deployed. You can check out the one I wrote for Shopify Shipit here: https://github.com/Shopify/shipit/pull/120/files#diff-af16ffbedffc7b5f9e26cab1dd292bee. 

Essentially, the kubeconfig contains a list of clusters (e.g. we'll have one for each service tier) and a list of user accounts (just "shipit" for us, but e.g. locally I have a service account and my google auth login) + a list of contexts that pair the two together. All the apps being deployed need to specify is the name of the context: https://github.com/KnVerey/trashbin/blob/master/shipit.yml. 

I don't feel strongly about this, but the reason I left it as an env var is that that seemed to be the best way to provide it from Shipit to the script (since it is always the same in a given Shipit deployment) and `kubectl` supports using it directly: http://kubernetes.io/docs/user-guide/sharing-clusters/#merging-kubeconfig-example. There is a standard path, but it's `~/.kube/config`.